awx/main/utils/filters.py

```diff
@@ -364,3 +367,11 @@ def query_from_string(cls, filter_string):
             return res[0].result
 
         raise RuntimeError("Parsing the filter_string %s went terribly wrong" % filter_string)
+
+
+
+class DefaultCorrelationId(CorrelationId):
+
+    def filter(self, record):
+        record.correlation_id = GuidMiddleware.get_guid() or '-'
```

Comment:
Slight modification - I didn't like that the default filter printed `[None]` in various places.